Bonsai.Tinkerforge/DeviceData.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bonsai.Tinkerforge
+{
+    /// <summary>
+    /// Data representation of a connected TinkerForge module
+    /// </summary>
+    public class DeviceData
+    {
+        public string Uid; // Unique module ID
+        public string ConnectedUid; // IDs of connected modules
+        public char Position; // Position in the network
+        public int DeviceIdentifier; // Number corresponding to device name
+
+        // Parameterless constructor required for serialization
+        public DeviceData()
```

Comment:
Since we don't seem to be serializing this anymore, do we still need the parameterless constructor?